docs/reference/config.rst

```diff
@@ -670,6 +670,16 @@ with the ``-a`` flag to the :ref:`import-cmd` command.)
 
 Default: ``yes``.
 
+.. _duplicate_keys:
+
+duplicate_keys
+~~~~~~~~~~~~~~
+
+The fields used to find duplicates in import task.
+If several items have the same value for each key, they will be considered duplicates.
```

Comment:
It occurs to me here that we search for duplicates for both albums and items, and maybe we want the list of fields to be configurable separately.

Reply:
Actually it should use album keys only